docs/config.rst

```diff
@@ -1046,6 +1046,16 @@ Marks
 
    See also: :attr:`LP_ENABLE_HG` and :attr:`LP_HG_COMMAND`.
 
+.. attribute:: LP_MARK_JOBS_SEPARATOR
+   :type: string
+   :value: "/"
+
+   Mark used to separate elements of :attr:`LP_JOBS`.
+
+   See also :attr:`LP_ENABLE_JOBS`.
+
+   ..versionadded:: 2.2
```

Comment:
Missing space after `..` apparently breaks rendering. I think the docs-lint might catch that.

Reply:
It did:
>  _build/html/config.html:1238: possibly leaked markup: \<p>..versionadded:: 2.2\</p>

Reply:
It did.